common/names.go

```diff
@@ -0,0 +1,6 @@
+package common
+
+// Defins well-known names in the A01 system
+const (
+	NameStorageVolume = "azure-storage"
```

Comment:
>NameStorageVolume [](start = 1, length = 17)

Golint (IIRC) requires to add documentation for exported stuff.

Reply:
I thought a definition on a block is enough.

Reply:
There are many cases in which comments on each exported constant value becomes verbose. For example in the `data.go` which I define the reserved property values.

I ran `golint` didn't see these error after I comment on the block (line 3). Did I not run it correctly?

Reply:
Nope, you are good :) Sorry, I got used to the extreme amount of comments the go sdk can have

Reply:
I think whoever wrote the `http` package felt the same way:
```
// Copyright 2015 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

package http

// Common HTTP methods.
//
// Unless otherwise noted, these are defined in RFC 7231 section 4.3.
const (
	MethodGet     = "GET"
	MethodHead    = "HEAD"
	MethodPost    = "POST"
	MethodPut     = "PUT"
	MethodPatch   = "PATCH" // RFC 5789
	MethodDelete  = "DELETE"
	MethodConnect = "CONNECT"
	MethodOptions = "OPTIONS"
	MethodTrace   = "TRACE"
)
```